src/extension/app/store/app.js

```diff
@@ -288,7 +293,7 @@ export class AppStore {
               text: (titleI18n && titleI18n[lang]) || title,
               action: () => {
                 if (url) {
-                  const target = url.startsWith('/') ? new URL(url, `https://${innerHost}/`) : new URL(url);
+                  const target = devMode ? new URL(url, devUrl) : new URL(url, `https://${innerHost}/`);
```

Comment:
Shouldn't you be able to open the palette iframe on another origin?

Reply:
Yes, you can: in case of a full URL, the second argument of the `URL()` constructor will be ignored.